### DISCUSSION TITLE
请问"全局设置-访问控制"下完全不走clash的IP或MAC这个功能去哪里了呢?

### DISCUSSION THREAD

**Discussion by Mosney** [vernesong/OpenClash#2505] [Q&A] [answered]
安装了v0.45.33发现之前设置的不走clash的局域网IP `10.0.0.4`发起的连接也出现在clash日志和连接中，进openclash才发现这个功能不见了，取而代之的成了"不走代理的 WAN IP"，百思不得其解。尝试把`10.0.0.4/30` 添加到这里，应用后发现并不生效，看起来之前完全不进入clash的设备黑白名单功能彻底没有了？ @vernesong

**Comment by Mosney** [comment, marked as answer]
自问自答：
刚想起来前不久redir-host模式切换到fake-IP模式了。如果完全不进入clash设备的DNS不做额外处理的话，结果就是设备拿到的IP全是`198.18.0.1/16`这种IP，不走clash的话这些向不存在于公网上的IP发起的连接就会全都失败，进而导致对应设备不能上网。

**Reply by kid101x** [reply]
请问最后如何解决这个需求的，比如BT设备绕过OpenClash。

**Reply by Mosney** [reply]
https://github.com/vernesong/OpenClash/issues/2506#issuecomment-1178942320
